.github/workflows/ci.yml

```diff
@@ -7,8 +7,36 @@ on:
     branches: [ main ]
 
 jobs:
+  check_nuspec:
+    runs-on: windows-2016
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Get changed files
+        id: files
+        uses: Ana06/get-changed-files@v2.0.0
+        with:
+          filter: 'packages/*/**'
+      - name: Check nuspec is modified for every modified package
+        # It runs only if there are modified files
+        if: steps.files.outputs.added_modified != ''
+        run: |
+          $changed_files = "${{ steps.files.outputs.added_modified }}".Split(" ")
+          $nuspecs = @($changed_files | where { $_ -Like "*.nuspec" })
+          $others = @($changed_files | where { $_ -NotLike "*.nuspec" })
+          foreach ($file in $others) {
+              if ($file -match "packages/([^/]*)") {
+                  $package = $matches.1
+                  $nuspec = "packages/$package/$package.nuspec"
+                  if (!$nuspecs.contains($file)) {
```

Comment:
I think the path `"packages/$package/$package.nuspec"` may not always hold true (e.g., see path for [notepadpp.vm's .nuspec file](https://github.com/mandiant/VM-Packages/blob/main/packages/notepadpp.vm/notepadplusplus.vm.nuspec)). Also wouldn't it be easier to check if a `$nuspecs` is not empty?

Reply:
> I think the path "packages/$package/$package.nuspec" may not always hold true (e.g., see path for notepadpp.vm's .nuspec file)

isn't that nuspec name wrong? I think the directory and the nuspec should have the same name. 🤔 

> Also wouldn't it be easier to check if a $nuspecs is not empty?
we could. But if you edit two packages and only one nuspec is updated, then the action would work. Ideailly we should check that the version has been correctly updated. Remember that if the action success in a PR merge, the package is pushed to myget.

Reply:
One easier alternative would be to keep $nuspecs as string and check it contains `packages/$package/`. This may be a more robust solution. 🤔

Reply:
I like the new version 😄